bdp_app/lib/pages/ChatPage.dart

```diff
@@ -86,8 +106,15 @@ class _ChatPageState extends State<ChatPage> {
           username: widget.username,
           sendername: message["senderName"],
           destination: widget.username,
-          chatID: message["senderName"],  // todo because chatMessages are local
-          text: message["text"]);
+          messageId: message["id"].toString(),
+          chatID: message["senderName"],
+          // todo because chatMessages are local
+          groupId:
+              (message["groupId"] == null) ? "" : message["groupId"].toString(),
+          text: message["text"],
+          sentTime: DateTime.now().toString().substring(
+              11, 16) //todo: this is actually not sentTime. do we want that?
```

Comment:
Können wir hier nicht einfach ein DateTime-Object benutzen und das erst beim Darstellen auf dem Screen zu einem String formatieren? Dann speichern wir wesentlich konkretere Informationen und sind flexibel in der Darstellung.